src/webrtc/groupCall.ts

```diff
@@ -408,6 +414,12 @@ export class GroupCall extends EventEmitter {
 
         // set a timer for the maximum transmit time on PTT calls
         if (this.isPtt) {
+            // if anoher user is currently unmuted, we can't unmute
```

Comment:
This seems a bit weird to force on everyone.

If I was using this for game callouts, I would want to interrupt the other person.